stdlib/http/http_path_encode_endpoint_test.flux

```diff
@@ -0,0 +1,39 @@
+package http_test
```

Comment:
I wonder if we really need an endpoint test for this. From my understanding of this test, it tests the same code path as the unit test while also using a lot of overhead that doesn't contribute to the test. It might make good example code for how to use `http.pathEscape`, but as a test, it seems unnecessary. Is that off base?

Reply:
From speaking with Chris, we thought it might be good to see how this test functions with everything else. If this causes bigger issues it might make sense to revisit its necessity.

Reply:
I'm normally a little reticent to have multiple tests testing the same thing (they get hard to remove later), but if you and @wolffcm have talked it over and feel confident this is right, I'm happy for it to merge.

Reply:
Yeah, my reasoning is that functions executed from inside of a table stream function like `map` are using a slightly different code path, and so it's worth adding tests for that case. For example, the `Context` might have different stuff in it, like we saw with functions that needed to access `now`. 

In general I totally agree that unneeded testing can become painful if left unchecked.